### PR TITLE
Added new entries for multiple companies

### PR DESCRIPTION
**Pull request by upgundecha**
Added new posts from -

* ASOS
* Canva
* Disney+ Hotstar
* Expedia
* Goibibo
* Helpshift
* Mercari

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -185,6 +186,7 @@ Testing / quality of software goes beyond traditional activities of pre-release
   <summary>Canva</summary>
 
 #### Blogs & Articles
+* [Why we left manual UI testing behind](https://product.canva.com/automated-visual-testing-at-canva/)
```
Comment (abhivaikar) [inline README.md:189]: this is the duplicate of the one below

File: README.md [hunk 2]
```diff
@@ -341,6 +345,9 @@ Testing / quality of software goes beyond traditional activities of pre-release
   <summary>Expedia</summary>
 
 #### Blogs & Articles
+* [How Expedia.com Uses Accessibility Automation](https://medium.com/expedia-group-tech/how-expedia-com-uses-accessibility-automation-d3de3e3ff775)
+* [Improving iOS UI Testing](https://medium.com/expedia-group-tech/improving-ios-ui-testing-ad55470825b4)
+[Unit Testing Apache Spark Applications using Hive Tables](https://medium.com/expedia-group-tech/unit-testing-apache-spark-applications-using-hive-tables-ec653c6f25be)
```
Comment (abhivaikar) [inline README.md:350]: This needs to be a bullet point

File: README.md [hunk 3]
```diff
@@ -456,6 +463,7 @@ Testing / quality of software goes beyond traditional activities of pre-release
   <summary>Goibibo</summary>
 
 #### Blogs & Articles
+* [Mr. Postman for Integration Testing](https://tech.goibibo.com/mr-postman-for-integration-testing-879a4031f916)
```
Comment (abhivaikar) [inline README.md:466]: Duplicate. The link is already there.

File: README.md [hunk 4]
```diff
@@ -543,17 +551,18 @@ Testing / quality of software goes beyond traditional activities of pre-release
 <details>
   <summary>Helpshift</summary>
 
-#### Videos
-* [Testing business logic using DSLs in Clojure](https://youtu.be/YOsfPrgNY_M)
-* [Clojure for functional testing of Mobile and Web apps](https://youtu.be/G-fjpO6nYPQ)
-
 #### Blogs & Articles
+* [The Convoluted Magic of Leiningen Test Selectors](https://medium.com/helpshift-engineering/the-convoluted-magic-of-leiningen-test-selectors-2eb6c452dfcf)
```
Comment (abhivaikar) [inline README.md:555]: Duplicated.

**Review by abhivaikar — CHANGES_REQUESTED** [review]

File: README.md (hunk 4 above)
Comment (upgundecha) [reply README.md:555]: Removed

File: README.md (hunk 3 above)
Comment (upgundecha) [reply README.md:466]: Bad search in my IDE :(

File: README.md (hunk 2 above)
Comment (upgundecha) [reply README.md:350]: Fixed

File: README.md (hunk 1 above)
Comment (upgundecha) [reply README.md:189]: Ah title. Fixed

**Review by abhivaikar — APPROVED** [review]

**Pull request merged**